<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Createkey</class>
 <widget class="QDialog" name="Createkey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create key-pair</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Expire :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Comment (optionel) :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Your email :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Your Name (minimum of 5 chars :) :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="lineComment">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEmail">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="lineName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="checkExpire">
        <property name="text">
         <string>No expiration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2013</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="date">
         <date>
          <year>2013</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="comboLength">
        <item>
         <property name="text">
          <string>1024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4096</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboAlgo">
        <item>
         <property name="text">
          <string>RSA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DSA</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Passphrase :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Primary key :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Key type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboAlgo2">
        <item>
         <property name="text">
          <string>RSA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ELG-E</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboLength2">
        <item>
         <property name="text">
          <string>1024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4096</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Subkey :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="linePassphrase">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Key size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
